lib/atoms/title1/directive.js

```diff
@@ -0,0 +1,15 @@
+angular
```

Comment:
Yes, we can do that but have to adapt gulp build tasks to consider the change because they are based on the names conventions. I'll include that change in the future commits.